software/oort-config/role/firewall/_MANIFEST.csv

```diff
@@ -2,5 +2,18 @@
 USER,GROUP,PERM,ABSPATH
 
 # Preexisting directories - not actual overrides
+root,wheel,755,/etc
+root,wheel,755,/var
+root,wheel,755,/var/unbound
+root,wheel,755,/var/unbound/etc
 
 # OVERRIDES START
+root,wheel,644,/etc/dhcpd.conf
+root,wheel,640,/etc/hostname.re0.conf
```

Comment:
I think you have an unintentional ".conf" suffixed on the three hostname.* files (this will be a lot easier to catch once I actually give you the tool to run!)

Reply:
Nice catch! Fixed